Assets/Scripts/Room/RoleSelectManager.cs

```diff
@@ -51,6 +54,20 @@ private bool CanStartGame()
         return selectedRoles.Count == System.Enum.GetValues(typeof(PlayerType)).Length;
     }
 
+    private void IndicatorUpdate(Player player, PlayerType type)
+    {
+        if (player == PhotonNetwork.LocalPlayer)
+        {
+            yourRoleIndicator[(int) type].SetActive(true);
+            yourRoleIndicator[1-(int) type].SetActive(false);
```

Comment:
Perhaps it'd be better to create a new `RoleIndicator` class that holds a `Dictionary<PlayerType, GameObject>` so that we don't have to rely on the individual numeric values of the `PlayerType` enum.